docs/astro/src/content/docs/guide/language/concepts/slint-language.mdx

```diff
@@ -48,16 +48,16 @@ document.body.appendChild(buttonWithListener);
 ```
 
 In this second example a button is created and an event listener is added. Within that is a callback
-function to log out that the button is pressed. It's a lot of code to so simple things.
+function to log out that the button is pressed. It's a lot of code to do simple things.
 
-Its quite abstract. For example once a few more buttons and components are added its almost impossible
+It's quite abstract. For example once a few more buttons and components are added its almost impossible
 to be able to think how the real interface would look. It's hard to think about the design of a UI
 using this kind of code.
 
 It's also too complex to edit without understanding how to code. This means UI designers cannot work
 hands on to ensure all their design intent is implemented. They are forced to use other tools and frameworks
-to crate prototypes and design guides that may look or behave differently to the software used to implement
-the production UI. It should not have to be this way.
+to create prototypes and design guides that may look or behave differently to the software used to implement
+the production UI. It doesn't have to be this way.
```

Comment:
`the software used to implement the production UI` is a bit awkward. Can I change it to `the actual UI implementation`?

Reply:
Please do change that